### PR TITLE
Second Pass

### PR DESCRIPTION
**Pull request by cpaasch**

### CODE REVIEW

File: draft-ietf-mptcp-rfc6824bis-11.xml [hunk 1]
```diff
@@ -1430,26 +1430,24 @@ break the multipath connections, it will just reduce the opportunity to open mul
...
-	    <t>Upon receipt of a RST with MP_FASTCLOSE, containing the valid key,
-	    Host B tears down all subflows. Host B can now close the whole MPTCP
-	    connection (it transitions directly to CLOSED state).</t>
           </list>
         </t>
+
+	<t>If however host A decides to force the closure by using Option R and
+	sending a RST with the MP_FASTCLOSE option, Host B will act as follows:
+	Upon receipt of a RST with MP_FASTCLOSE, containing the valid key,
+	Host B tears down all subflows. Host B can now close the whole MPTCP
```
Comment (ajf101) [inline draft-ietf-mptcp-rfc6824bis-11.xml]: Should we require a RST from Host B on all subflows too?

File: draft-ietf-mptcp-rfc6824bis-11.xml [hunk 2]
```diff
@@ -2050,7 +2048,7 @@ subflow-specific windows.</t>
         <c>This document, <xref target="sec_init"/></c>
 
         <c>C</c>
-        <c>Do not attempt to connect to source address</c>
+        <c>Don't reuse source address for new subflows.</c>
```
Comment (ajf101) [inline draft-ietf-mptcp-rfc6824bis-11.xml]: That wording is not very clear to me. How about:

"Do not attempt to establish new subflows to the source address"

**Review by ajf101 — COMMENTED** [review]
Two comments included

File: draft-ietf-mptcp-rfc6824bis-11.xml (hunk 1 above)
Comment (cpaasch) [reply draft-ietf-mptcp-rfc6824bis-11.xml]: Might be useful! I will add this to the text.

File: draft-ietf-mptcp-rfc6824bis-11.xml (hunk 2 above)
Comment (cpaasch) [reply draft-ietf-mptcp-rfc6824bis-11.xml]: Ok, used your suggestion.

**Comment by cpaasch** [comment]
@ajf101 , your comments have been addressed.

**Pull request merged**